tests/nnstreamer_filter_extensions_common/meson.build

```diff
@@ -48,6 +48,10 @@ if have_python3
 endif
 
 sed_command = find_program('sed', required: true)
+sed_command_edit_in_place_option = '-i'
+if build_platform == 'macos'
+  sed_command_edit_in_place_option = '-i .bak'
```

Comment:
macOS sed can be told to edit in place without writing an additional backup file, using

```sh
sed -i '' s/foo/bar/ file.txt
```